docs/tutorial/test_model.rst

```diff
@@ -11,19 +11,32 @@ you will see the interface of the toolbox:
 
 .. image:: ../pictures/ModelTest1.jpg
 
-Use the Properties Window (1) to choose the RodPendulumModel, 
-the ODEInt Solver and the AdditiveMixer as ModelMixer. 
+Within the Properties Window (1), double clicking on a value (all :py:data:`None` by default)
+activates a drop down menu.
+Clicking again presents all eligible options.
+One of these options as an eligible model is the class RodPendulumModel,
+since it was registered and linked to PyMoskito earlier.
+Choose it now and press enter to confirm your choice.
+
+By clicking on the arrow that appeared on the left of :py:data:`Model`,
+all model parameters and the initial state are displayed. 
+These are taken from the public part of the ordered dictionary defined earlier in the model.
```

Comment:
Maybe something around the lines of "These are taken from the `public_settings` which have been defined earlier.